engine/src/main/java/org/destinationsol/ui/nui/screens/MainGameScreen.java

```diff
@@ -55,8 +54,16 @@
     private UIWarnButton mercsButton;
     private ConsoleScreen consoleScreen;
 
-    @In
-    private SolApplication solApplication;
+    @Inject
+    protected SolApplication solApplication;
+
+    public MainGameScreen() {
+
+    }
+
+    public void setSolApplication(SolApplication application) {
```

Comment:
Are NUI screens injected by `gestalt-di` yet? If so, then why is this method needed?

Reply:
there is a problem here so I kind of have a hacky workaround.